src/models/installation.ts

```diff
@@ -142,12 +143,14 @@ Installation.init({
 	hooks: {
 		beforeSave: async (instance: Installation, opts) => {
 			if (!opts.fields) return;
-			await instance.encryptChangedSecretFields(opts.fields, LOGGER_HOOK_BEFORE_SAVE);
+			const optsFields = opts.fields.filter((it): it is string => !!it);
```

Comment:
Curious why do we need this change?

Reply:
With the beefed up type checking in sequelize v6 we now need to be more explicit here. We're setting `fieldsChanged: string[] = []` in `encryptChangedSecretFields` and without this update, in `beforeSave` and `beforeBulkCreate` we get the following error:

<img width="670" alt="Screenshot 2023-03-31 at 9 44 59 am" src="https://user-images.githubusercontent.com/37155488/228981603-bdbee9ff-8474-4210-ba0d-27161bc4aa46.png">

This filters out any values that aren't of type string.